.github/actions/spelling/action.yml

```
@@ -27,15 +27,16 @@ inputs:
     type: string
     default: .github/actions/spelling
   spell-check-this:
-    description: Repository with default configuration to use
+    description: Repository with default configuration to use, the default from Check Spelling is ''
```

Same for the workflow.

```suggestion
    description: Repository with default configuration to use, the default from Check Spelling is ''.
```